docs/index.rst

```diff
@@ -86,6 +86,7 @@ anonymous sources.
    :name: upgradetoc
    :maxdepth: 2
 
+   upgrade/0.12.2_to_0.13.0.rst
```

Comment:
Was the intent for this link to be on the top? 0.12.1->0.12.2 was put below the Xenial upgrade step, despite being released after the Xenial Docs

Reply:
Yes, I figure we don't want to permanently keep the Xenial related docs at the top now that the upgrade is complete. It made sense to me to group them together before then. Happy to consider alternative ways to organize this though!